splipy/utils/__init__.py

```diff
@@ -9,6 +9,34 @@
 except ImportError:
     from collections import Sized
 
+def is_right_hand(patch, tol=1e-3):
+    if patch.dimension == patch.pardim == 3:
+        param = patch.start()
```

Comment:
Suggest one sample this in the patch midpoint instead of one of one of the corners. Endpoints might correspond to degenerate sides which would then fail the test.